spec/mongoid/enum_spec.rb

```diff
@@ -174,4 +192,21 @@ class User
       end
     end
   end
+
+  describe ".configuration" do
+    it "returns Configuration object" do
+      expect(Mongoid::Enum.configuration)
+          .to be_instance_of Mongoid::Enum::Configuration
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 4) spaces for indenting an expression spanning multiple lines.